Tasks/DotNetCoreCLI/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -21,6 +21,8 @@
   "loc.input.help.arguments": "Arguments to the selected command. For example, build configuration, output folder, rutime. The arguments depend on the command selected.",
   "loc.input.label.zipAfterPublish": "Zip Published Projects",
   "loc.input.help.zipAfterPublish": "If true, folder created by the publish command will be zipped.",
+  "loc.input.label.modifyOutputPath": "Add project name to publish path",
+  "loc.input.help.modifyOutputPath": "If true, folders created by the publish command will have project file name prefixed to their folder names when output path is specified explicitly in arguments.",
```

Comment:
mention that htis is useful if you want to publish multiple projects to the same folder.